### ISSUE TITLE
Return detailed diff from the `azure_rm_aks` module

### ISSUE THREAD

**Issue by emanueleaina** [ansible/ansible_collections_azure#1534]
##### SUMMARY
The `azure_rm_aks` module at the moment only reports if something has changed, but it does not yield any detailed information when running in `--diff` mode.

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME
`azure_rm_aks`

##### ADDITIONAL INFORMATION
Given that the number of parameters is large, finding exactly what is differing requires a lot of guesswork: for instance at some point I ended up hacking the module to print which key in the `network_profile` was actually differing.

**Comment by Fred-sun** [comment]
@em- I am very sorry that the collection azure. azcollection does not support diff mode. However, we can get the current resource information using the *****_info.py module. Then consider whether to update, configured in the parameters of the ****.py module. Thank you! thank you!

**Comment by emanueleaina** [comment]
Yep, at the moment I am collecting the info and passing it to the `fact_diff` module, but it is all quite ugly. :)
